src/insns/csrr_32bit.adoc

```diff
@@ -30,31 +30,49 @@ endif::[]
 Synopsis::
 CSR access (CSRRWI, CSRRS, CSRRSI, CSRRC, CSRRCI) 32-bit encodings
 
-Register Source Mnemonics::
-`csrr[s|c] rd, rs1, csr`
-
-Immediate Source Mnemonics::
-`csrr[w|s|c]i rd, imm, csr`
+Mnemonics for accessing capability CSRs at CLEN-wide aliases::
+`csrrs  cd, rs1, csr` +
+`csrrc  cd, rs1, csr` +
+`csrrwi cd, imm, csr` +
+`csrrsi cd, imm, csr` +
+`csrrci cd, imm, csr`
+
+Mnemonics for accessing XLEN-wide CSRs or capability CSRs at XLEN-wide aliases::
+`csrrs  rd, rs1, csr` +
+`csrrc  rd, rs1, csr` +
+`csrrwi rd, imm, csr` +
+`csrrsi rd, imm, csr` +
+`csrrci rd, imm, csr`
 
 Encoding::
 include::wavedrom/csr-instr.adoc[]
 
 Description::
 These are standard RISC-V CSR instructions with extended functionality for
-accessing CLEN-wide CSRs, such as <<mtvec>>/<<mtvecc>> which can be
+accessing CLEN-wide CSRs, such as <<mtvec>>/<<mtvecc>>, which can be
 accessed through either the RISC-V address or the capability address alias.
 +
-Unlike <<CSRRW>> these instruction perform the same update to CLEN-wide CSRs
- to either the XLEN or CLEN-wide alias as they only every perform an XLEN-wide update.
- Where a CLEN-wide CSR is updated, through either alias, the final address is determined
- as defined by RISC-V for these instructions. The metadata and tag are updated
- as defined in xref:extended_CSR_writing[xrefstyle=short].
+Unlike <<CSRRW>>, these instructions only update the address field when writing
+capability CSRs regardless of the CSR's alias used. The final address to write
+to the capability CSR is determined as defined by RISC-V for these
+instructions. The metadata and tag are updated as defined in
+xref:extended_CSR_writing[xrefstyle=short].
 +
 See xref:aliased_CSRs[xrefstyle=short] for a list of CLEN-wide CSRs and
 xref:extended_CSR_writing[xrefstyle=short] for the action taken on writing an XLEN-wide value to each one.
 +
+If `cd` is `c0` (or `rd` is `x0`), then <<CSRRWI>> shall not read the CSR and
```

Comment:
Should we add a note here that`csrr cd, csr` maps to csrs to match the base ISA?

Reply:
I made a note about csrr and csrw.